### ISSUE TITLE
SF Display sliders crash program

### ISSUE THREAD

**Issue by gordonnish** [CloudCompare/CloudCompare#938]
I'm running v2.10.2 Stereo 64 bit. Quite often, when I move the slider for display ranges or saturation for my cloud, I find that the program crashes on me and quits to the desktop. I have searched all over for another instance of this issue, and haven't found it yet. Let me know if you would like me to upload a movie of it, or crash log (if that exists) or anything else to help out. Thanks.

**Comment by dgirardeau** [comment]
I believe it can happen with some specific graphic cards... Which one do you have?

Can you try to disable the 'Color scale(scalar field) > Enable shader for faster display...' option in the 'Display options' dialog? And then see if you manage to reproduce this crash?

**Comment by gordonnish** [comment]
I was able to reproduce it a few times after following your instructions. I am running a 1080ti.

**Comment by dgirardeau** [comment]
Interesting... Can you maybe share the data with me (admin [at] cloudcompare.org) and maybe a video? So I can try to reproduce it on my side.

**Comment by gordonnish** [comment]
Did you receive the data/video?

**Comment by dgirardeau** [comment]
Yes thanks. I am just back from vacations, I'll review that asap.

**Comment by dgirardeau** [comment]
Ok I looked at the video, downloaded the data, and tested it on my side. I sadly can't reproduce the issue, but one culprit may be (at least on your side) the LOD mechanism.

Can you try on your side the same procedure by first deactivating it? ("Display > Display Settings", then uncheck the "decimate clouds over..." checkbox in the "Other options" tab).

**Comment by gordonnish** [comment]
That looks like it did the trick. It's an unfortunate thing though, as now it's much less smooth when rotating the cloud (obviously). Thanks for the help!

**Comment by dgirardeau** [comment]
The issue is not fixed ;)

We just found the culprit (the LOD mechanism), but we should fix it now ;)

**Comment by gordonnish** [comment]
By all means, fix away. I would help out if I could, but coding is not my strong suit. Continuez avec le bon travail et bon courage aussi.

**Comment by dgirardeau** [comment]
Additional question: is the option 'Enable shader for faster display (ATI cards:...)' enabled on your side (see the 'Color scale (scalar field)' tab in the Display options)

**Comment by gordonnish** [comment]
No that was not enabled when the crashes occurred.

**Comment by dgirardeau** [comment]
And if you activate it (with the LOD mechanism activated as well), are you
able to reproduce the crash?

Le lun. 26 août 2019 à 15:16, gordonnish <notifications@github.com> a
écrit :
